CONTRIBUTING.md

```diff
@@ -295,3 +295,79 @@ You can find an installation guide for all the different environments here:
+- <span style="color:#5319E7">**state: tbd**</span>: If the issue has the state tbd (to be defined) it means the team does need more information either from the author or internal.
+- <span style="color:#5319E7">**state: backlog**</span>: If an issue is in the backlog, it is not currently being worked on. These are recorded so that they can be processed in the future. Issues with this state do not have to be completely defined yet.
+- <span style="color:#5319E7">**state: ready**</span>: An issue with the state ready is ready to implement. This means the developer can find all the relevant information and acceptance criteria in the issue.
+- <span style="color:#5319E7">**state: in progress**</span>: Someone is working on this issue right now.
+- <span style="color:#5319E7">**state: waiting**</span>: For some reason, this issue will have to wait. This can be a feedback that is being waited for, a dependent issue or anything else.
+- <span style="color:#5319E7">**state: duplicate**</span>: The same issue already exists. This issue will probably be closed with a reference to the other issue.
+
+### Category
+
+The category shows which part of ZITADEL is affected.
+
+- <span style="color:#006B75">**category: backend**</span>: The backend includes the APIs, event store, command and query side. This is developed in golang.
+- <span style="color:#006B75">**category: ci**</span>: ci is all about continues integration and pipelines.
+- <span style="color:#006B75">**category: design**</span>: All about the ux/ui of ZITADEL
+- <span style="color:#006B75">**category: docs**</span>: Adjustments or new documentations, this can be found in the docs folder.
+- <span style="color:#006B75">**category: frontend**</span>: The frontend concerns on the one hand the ZITADEL management console (Angular) and on the other hand the login (gohtml)
+- <span style="color:#006B75">**category: infra**</span>: Infrastructure does include many different parts. E.g Terraform-provider, docker, metrics, etc.
+- <span style="color:#006B75">**category: translation**</span>: Everything concerning translations or new languages
+
+### Language
+
+The language shows you in which programming language the affected part is written
+
+- <span style="color:#0052CC">**lang: angular**</span>
+- <span style="color:#0052CC">**lang: go**</span>
+- <span style="color:#0052CC">**lang: javascript**</span>
+
+### Effort
+
+The effort should give you an indication how much work it takes. This is based on a rough estimation.
+Everything that is higher than 8 should be split in smaller parts.
+
+- <span style="color:#BFDADC">**effort: 1**</span>
+- <span style="color:#BFDADC">**effort: 2**</span>
+- <span style="color:#BFDADC">**effort: 3**</span>
+- <span style="color:#BFDADC">**effort: 5**</span>
+- <span style="color:#BFDADC">**effort: 8**</span>
+
+
+
+
+
+
+
```

Comment:
Lets remove those. Maybe run a mdlint across the doc

Reply:
I fixed most of the linting problems. Only some inline html is still in the file.